docs/docs/guides/smart_contracts/writing_contracts/how_to_prove_history.md

```diff
@@ -5,11 +5,13 @@ sidebar_position: 4
 
 The Aztec Protocol uses an append-only Merkle tree to store hashes of the headers of all previous blocks in the chain as its leaves. This is known as an archive tree. You can learn more about how it works in the [concepts section](../../../aztec/concepts/storage/trees/index.md#archive-tree).
 
-View the History lib reference [here](../../../reference/smart_contract_reference/history_lib_reference.md).
+This page is a quick introductory guide to creating historical proofs proofs from the archive tree.
 
-# History library
+For a reference, go [here](https://docs.aztec.network/reference/smart_contract_reference/aztec-nr/aztec/history/contract_inclusion).
 
-The history library allows you to prove any of the following at a given block height before the current height:
+# Inclusion and non-inclusion proofs
```

Comment:
```suggestion
## Inclusion and non-inclusion proofs
```
because otherwise it looks weird with text above the top-level heading 
![image](https://github.com/user-attachments/assets/d3fc0225-67be-41d6-8e00-9c0b4758f36f)